bin/update_submodules.sh

```diff
@@ -4,9 +4,10 @@
 git submodule update --init --recursive
 
 # update monero-cpp
+# needs to be uncommented and changed to a release tag when release 
```

Comment:
These comment can be removed since the lines below will checkout and pull master whereas release tags will checkout and pull a specific version.